package.json

```diff
@@ -89,6 +89,7 @@
     "nodemon": "^3.0.2",
     "sinon": "^17.0.1",
     "ts-jest": "^29.1.1",
+    "ts-node": "^10.9.2",
```

Comment:
why do we need ts-node?

Reply:
An error message told me it was required after I switched jest.config.js to jest.config.ts

Reply:
I removed it and I'm not getting the error now so maybe it'll work without it